google/iam/admin/v1/iam_gapic.yaml

```diff
@@ -0,0 +1,238 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.iam.admin.spi.v1
+  python:
+    package_name: google.cloud.gapic.iam.admin.v1
```

Comment:
toolkit is throwing an invalid package name error when using iam-admin. I am going to use google.cloud.gapic.iam_admin.v1 instead.